src/Xamarin.Android.Build.Tasks/Utilities/LlvmIrGenerator/StringHolder.cs

```diff
@@ -0,0 +1,124 @@
+using System;
+
+namespace Xamarin.Android.Tasks.LLVMIR;
+
+class StringHolder : IComparable, IComparable<StringHolder>, IEquatable<StringHolder>
```

Comment:
any reason not to use a `record`?  You'll get `IEquatable<T>` for free!

Reply:
No reason, but I don't want to change it anymore :)